lib/google_sign_in/redirect_protector.rb

```diff
@@ -9,7 +9,7 @@ class Violation < StandardError; end
     QUALIFIED_URL_PATTERN = /\A#{URI::DEFAULT_PARSER.make_regexp}\z/
 
     def ensure_same_origin(target, source)
-      if target =~ QUALIFIED_URL_PATTERN && origin_of(target) != origin_of(source)
+      if target.blank? || (target =~ QUALIFIED_URL_PATTERN && origin_of(target) != origin_of(source))
         raise Violation, "Redirect target #{target} does not have same origin as request (expected #{origin_of(source)})"
```

Comment:
Can we use `target.inspect` here so the error message is clearer when `target` is nil or blank?

Reply:
Ah, yes, that's better. Thank you!